pkgs/applications/virtualization/distrobox/default.nix

```diff
@@ -17,7 +17,7 @@ stdenvNoCC.mkDerivation (finalAttrs: {
   nativeBuildInputs = [ makeWrapper ];
 
   # https://github.com/89luca89/distrobox/pull/1080
-  patches = [ ./always-mount-nix.patch ];
+  patches = [ (lib.optional (lib.versionOlder finalAttrs.version "1.7.0.1") ./always-mount-nix.patch) ];
```

Comment:
This won't work as written; anyway, the patch (and file) should just be removed.

Reply:
Ok I removed it.